app/editor/src/features/admin/media-types/constants/strings.ts

```diff
@@ -0,0 +1,2 @@
+export const NEW_MEDIA_TYPE_ROUTE = '/admin/media/types/0';
```

Comment:
I'm not a big fan of these types of constants.  URL paths should be found in one place, not all over the place.  I realize you're essentially trying to use a constant to make it in one place.  But now there will be file here and there, and everywhere with paths.  I'd move this to the routes component folder and keep them all in one place.  

Related thought - if we're making components self contained, we wouldn't want these types of constants used anyway.  We'd have a prop with a default value in the component.  That way we can control it for each instances.

Reply:
Makes sense, I am getting flashbacks to the REDUX constants from before 😛